gui/wxpython/gis_set.py

```diff
@@ -624,16 +626,18 @@ def OnDownloadLocation(self, event):
         """
         Download location online
         """
-        location = download_location_interactively(self, self.gisdbase)
+        grassdatabase, location, user_mapset = download_location_interactively(
```

Comment:
`mapset` is probably a good enough name in this context.

Reply:
Name user_mapset changed to just mapset (last commit).